docs/products/postgresql/howto/use-pg-audit-logging.md

```diff
@@ -0,0 +1,337 @@
+- `source_service`: the name of an Aiven for PostgreSQL
+- `destination_service`: the name of an Aiven for OpenSearch service
+
+```bash
+curl --request POST \
+   --url https://api.aiven.io/v1/project/{project_name}/integration \
+   --header 'Authorization: Bearer REPLACE_WITH_YOUR_BEARER_TOKEN' \
+   --header 'content-type: application/json' \
+   --data
+      '{
+         "integration_type": "logs",
+         "source_service": "REPLACE_WITH_POSTGRESQL_SERVICE_NAME",
+         "destination_service": "REPLACE_WITH_OPENSEARCH_SERVICE_NAME",
+      }'
+```
+</TabItem>
+<TabItem value="3" label="Aiven CLI">
+#### Use Aiven CLI
+
+Use the [Aiven CLI](/docs/tools/cli) to create the service integration.
+
+```bash
+avn service integration-create --project $PG_PROJECT  \
+   -t logs                                            \
+   -s $PG_SERVICE_NAME                                \
+   -d $OS_SERVICE_NAME
+```
+
+After the service integration is set up and propagated to the service configuration, the
+logs are available in Aiven for OpenSearch. Each log record emitted by audit logging is
+stored in Aiven for OpenSearch as a single message, which cannot be guaranteed for
+external integrations such as [Remote Syslog](/docs/integrations/rsyslog).
+</TabItem>
+</Tabs>
+
+## Visualize your logs
+
+If your logs are available in Aiven for OpenSearch, use
+[OpenSearch Dashboards](/docs/products/opensearch/dashboards/get-started) to visualize the
+logs.
+
+To preview your audit logs in OpenSearch Dashboards, use the filtering tool: select
+`AIVEN_AUDIT_FROM`, set its value to `pg`, and apply the filter.
+
+<img src={AuditLogsOpenSearchDashboards} class="image"/>
+
+:::note
+If the index pattern in OpenSearch Dashboards had been configured before you enabled the
+service integration, the audit-specific AIVEN_AUDIT_FROM field is not available for
+filtering. Refresh the fields list for the index in OpenSearch Dashboards under
+**Stack Management** → **Index Patterns** → Your index pattern → **Refresh field list**.
+:::
+
+## Disable audit logging
+
+Disable audit logging by setting the `pgaudit.feature_enabled` parameter to
+`false` in your service's advanced configuration. Use the Aiven
+[console](https://console.aiven.io/), [CLI](/docs/tools/cli), or
+[psql](https://www.postgresql.org/docs/current/app-psql.html).
+
+<Tabs>
+<TabItem value="1" label="Aiven Console" groupId="group1" default>
+### Disable in Aiven Console
+
+:::important
+In the [Aiven Console](https://console.aiven.io/), you can disable audit logging on a
+service only. To disable it on a database or for a user, use
+[psql](https://www.postgresql.org/docs/current/app-psql.html).
+:::
+
+1. Log in to [Aiven Console](https://console.aiven.io/), and go to your organization
+   \> project > Aiven for PostgreSQL service.
+1. On the **Overview** page of your service, select **Service settings** from the sidebar.
+1. On the **Service settings** page, go to the **Advanced configuration** section
```

Comment:
I forgot about it elsewhere, but there is an icon available now for service settings. :)